tests/generator-Tests/Unit-Tests/CodeGeneratorTests.cs

```diff
@@ -10,9 +10,9 @@
 namespace generatortests
 {
 	[TestFixture]
-	class JavaInteropCodeGeneratorTests : CodeGeneratorTests
+	class XAJavaInteropCodeGeneratorTests : CodeGeneratorTests
```

Comment:
I believe this change -- which is basically the removal of the `JavaInteropCodeGeneratorTests` fixture -- is what's responsible for the [drop in the number of unit tests run](https://github.com/xamarin/java.interop/pull/909#pullrequestreview-821927950).

The primary reason for removing it was to reduce the number of failing tests, but maybe it won't be too hard to re-enable this fixture…